_docs/beta-payments.md

```diff
@@ -13,6 +13,6 @@ This document lists the required details that needs to be sent for each of our p
 **Note** Occasionally there might be issues with one of the corridors and we might have to disable them temporarily. Once youre onboarded we'll let you know if this happens. Also if you're trying to use a disabled corridor you're going to receive a validation error stating that the corridor is not active, and you won't be able to create new transactions there.
 </div>
 
-# Brazil
+# India
 
-{% include corridors/brl-bank.md recipient_type='individual' %}
\ No newline at end of file
+{% include corridors/inr-bank.md recipient_type='individual' %}
```

Comment:
Why are you removing Brazil?

Reply:
Brazil has been moved to the public individual payments page as per Ugochukwu's request... https://github.com/transferzero/api-documentation/pull/141/files

Reply:
![Screenshot 2023-02-03 at 12 45 34](https://user-images.githubusercontent.com/40522592/216606882-bd15bf49-5a0b-44b6-93a1-5b6d2bdcdac9.png)